source/features/hide-useless-comments.tsx

```diff
@@ -55,11 +55,16 @@ function unhide(event: React.MouseEvent<HTMLButtonElement>): void {
 	event.currentTarget.parentElement!.remove();
 }
 
+function init(): void {
+	hideUselessComments();
+}
+
 features.add({
 	id: 'hide-useless-comments',
 	description: 'Hide useless comments like "+1"',
 	include: [
-		features.isIssue
+		features.isIssue,
+		features.isPRConversation
```

Comment:
All the changes to this file should be undone. PRs were explicitly avoided because “👍” might be an acceptable response to PRs